### ISSUE TITLE
Print file paths as file URIs

### ISSUE THREAD

**Issue by eygraber** [PatilShreyas/compose-report-to-html#32]
The current output of running the Gradle task is:

```
Checking directory '/home/eli/workspace/portal/samples/kotlin-inject/android-app/build/compose_report/raw'
Checking file '/home/eli/workspace/portal/samples/kotlin-inject/android-app/build/compose_report/raw/android-app_release-module.json'
Checking file '/home/eli/workspace/portal/samples/kotlin-inject/android-app/build/compose_report/raw/android-app_release-composables.csv'
Checking file '/home/eli/workspace/portal/samples/kotlin-inject/android-app/build/compose_report/raw/android-app_release-composables.txt'
Checking file '/home/eli/workspace/portal/samples/kotlin-inject/android-app/build/compose_report/raw/android-app_release-classes.txt'
Compose Compiler report is generated: /home/eli/workspace/portal/samples/kotlin-inject/android-app/build/compose_report/index.html
```

If a `file://` URI is used I believe the IDE will linkify them so it could be opened in the browser.

**Comment by PatilShreyas** [comment]
Thanks for the suggestion @eygraber. Will definitely add this

**Comment by PatilShreyas** [comment]
Thanks, @eygraber for suggesting this improvement. This has been released in `v1.0.0`
